datafusion/core/src/datasource/file_format/csv.rs

```diff
@@ -1251,11 +1251,8 @@ mod tests {
             "+-----------------------+",
             "| 50                    |",
             "+-----------------------+"];
-        let file_size = if cfg!(target_os = "windows") {
-            30 // new line on Win is '\r\n'
-        } else {
-            20
-        };
+
+        let file_size = std::fs::metadata("tests/data/one_col.csv")?.len() as usize;
```

Comment:
I wonder how this test is passing on CI? In theory we run these tests on CI: 

https://github.com/apache/datafusion/actions/runs/10044145872/job/27758535504

but somehow this test does not appear to run 🤔